<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="cisummarizer.CheckstyleProblemCollectorErrorTest" tests="5" skipped="0" failures="0" errors="0" timestamp="2020-08-12T23:14:59" hostname="LAPTOP-631HSM73" time="0.452">
  <properties/>
  <testcase name="it_should_have_no_problems_but_have_errors_when_given_a_path_to_bad_file(String)[1]" classname="cisummarizer.CheckstyleProblemCollectorErrorTest" time="0.319"/>
  <testcase name="it_should_have_no_problems_but_have_errors_when_given_a_path_to_bad_file(String)[2]" classname="cisummarizer.CheckstyleProblemCollectorErrorTest" time="0.046"/>
  <testcase name="it_should_have_no_problems_but_have_errors_when_given_a_path_to_bad_file(String)[3]" classname="cisummarizer.CheckstyleProblemCollectorErrorTest" time="0.016"/>
  <testcase name="it_should_have_no_problems_but_have_errors_when_given_a_path_to_bad_file(String)[4]" classname="cisummarizer.CheckstyleProblemCollectorErrorTest" time="0.055"/>
  <testcase name="forInfinitest()" classname="cisummarizer.CheckstyleProblemCollectorErrorTest" time="0.002"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[something went south with src\test\resources\checkstyle-problem-collector\blorp
   here is the exception: java.nio.file.NoSuchFileException: src\test\resources\checkstyle-problem-collector\blorp
[Fatal Error] :1:1: Premature end of file.
something went south with src\test\resources\checkstyle-problem-collector\empty.xml
   here is the exception: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Premature end of file.
[Fatal Error] :1:1: Content is not allowed in prolog.
something went south with src\test\resources\checkstyle-problem-collector\malformed.xml
   here is the exception: org.xml.sax.SAXParseException; lineNumber: 1; columnNumber: 1; Content is not allowed in prolog.
]]></system-err>
</testsuite>
